Makefile

```diff
@@ -175,16 +182,22 @@ release-localkube: out/localkube checksum
 	gsutil cp out/localkube.sha256 $(LOCALKUBE_BUCKET)/$(LOCALKUBE_VERSION)/localkube-linux-amd64.sha256
 
 .PHONY: update-releases
-update-releases: 
+update-releases:
 	gsutil cp deploy/minikube/k8s_releases.json gs://minikube/k8s_releases.json
 
-out/localkube-image: out/localkube
+localkube-image: out/localkube
 	# TODO(aprindle) make addons placed into container configurable
 	docker build -t $(REGISTRY)/localkube-image:$(TAG) -f deploy/docker/Dockerfile .
 	@echo ""
 	@echo "${REGISTRY}/localkube-image:$(TAG) succesfully built"
 	@echo "See https://github.com/kubernetes/minikube/tree/master/deploy/docker for instrucions on how to run image"
 
+buildroot-image: $(ISO_BUILD_IMAGE) # convenient alias to build the docker container
```

Comment:
yes because $(ISO_BUILD_IMAGE) depends on the Dockerfile, this is a "way" to define aliases in Makefiles